pkgs/tools/inputmethods/fcitx5/with-addons.nix

```diff
@@ -12,7 +20,12 @@ symlinkJoin {
       --prefix FCITX_ADDON_DIRS : "$out/lib/fcitx5" \
       --suffix XDG_DATA_DIRS : "$out/share" \
       --suffix PATH : "$out/bin" \
-      --suffix LD_LIBRARY_PATH : ${lib.makeLibraryPath (lib.flatten (map (x: x.extraLdLibraries or []) addons))}
+      --suffix LD_LIBRARY_PATH : "${lib.makeLibraryPath (lib.flatten (map (x: x.extraLdLibraries or []) addons))}"
+
+    ${lib.optionalString (fcitx5-configtool != null)''
```

Comment:
When would `fcitx5-configtool` be `null`?

Reply:
`pkgs.fcitx5-with-addons.override { fcitx5-configtool = null; }`

I can override the package to disable the configtool.

Reply:
yes